index.html

```diff
@@ -15,305 +15,364 @@
-              </a>
-            </li>
-          </ul>
+        <div class="container">
+          <div class="title">
+            <h1>
+              I'm Ricardo <br />
+              Glad to see <br class="br" />
+              you!
+            </h1>
+          </div>
+          <div>
+            <p class="para">
+              I’m a software developer! I can help you build a product, feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+          <div>
+            <a href="#" class="connect">Let's connect</a>
+          </div>
+          <div class="cont">
+            <ul class="icons">
+              <li class="list1">
+                <a href="#">
+                  <img src="./img/github.svg" alt="github logo" />
+                </a>
+              </li>
+              <li class="list1">
+                <a href="#">
+                  <img src="./img/Linkedin.svg" alt="Linkedin logo" />
+                </a>
+              </li>
+              <li class="list1">
+                <a href="#">
+                  <img src="./img/twitter.svg" alt="Twitter logo" />
+                </a>
+              </li>
+              <li class="list1">
+                <a href="#">
+                  <img src="./img/medium.svg" alt="medium logo" />
+                </a>
+              </li>
+              <li class="list1">
+                <a href="#">
+                  <img src="./img/look.svg" alt=" logo" />
+                </a>
+              </li>
+            </ul>
+          </div>
         </div>
       </section>
       <ul class="works">
         <li class="card-w1">
-          <div class="image-section">
-            <a href="#">
-              <img src="./img/SnapshootPortfolio.png" alt="Image Portfolio" />
-            </a>
+          <div class="card-img">
+            <div class="image-section">
+              <a href="#">
+                <img
+                  src="./img/SnapshootPortfolio.png"
+                  alt="Image Portfolio"
+                  class="imgx"
+                />
+              </a>
+            </div>
           </div>
-          <div class="project">
-            <div class="titlep1">
-              <h2>Tonic</h2>
+          <div class="card-text">
+            <div class="project">
+              <div class="titlep1">
+                <h2>Tonic</h2>
+              </div>
+              <ul class="description">
+                <li class="snip1">Canopy</li>
+                <li class="snip2">• Back End Dev</li>
+                <li class="snip3">• 2015</li>
+              </ul>
+            </div>
+            <div class="para">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
```

Comment:
Kindly apply the given design as shown below

### Original Design
![right2](https://user-images.githubusercontent.com/58376020/121755195-dd73b980-cb16-11eb-9c6d-c2a5b2904d1a.png)